test/metabase/api/database_test.clj

```diff
@@ -237,7 +237,7 @@
                                                  :table_id           (id :categories)
                                                  :special_type       "id"
                                                  :name               "ID"
-                                                 :display_name       "Id"
+                                                 :display_name       "ID"
```

Comment:
If by "we" you mean me, then yes :)

I think "ID" is correct outside of programming contexts. https://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized

Fun fact: apparently it originally stood for "identity document"